src/index.html

```diff
@@ -1,17 +1,136 @@
+          <label class="form-field">
+            How old are You?
+            <input 
+              type="number" 
+              value="12" 
+              name=" How old" 
+              min="1" 
+              max="100"
+            >
+          </label>
+          <label class="form-field">
+            Full date of birth:
+            <input 
+              type="date" 
+              placeholder="mm/dd/yyyy" 
+              name="birth"
+            >
+          </label>
+          <label class="form-field">
+            I accept the term of the agreement
+            <input type="checkbox" name="accept">
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>Registration</legend>
+          <label class="form-field">
+            E-mail:
+            <input 
+              type="email" 
+              placeholder="email@example.com" 
+              name="E-mail" 
+              required
+            >
+          </label>
+          <label class="form-field">
+            Password
+            <input 
+              type="password" 
+              name="Password" 
+              required minlength="5" 
+              maxlength="32"
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>An interesting fact about you!</legend>
+          <div class="form-field">
+            Do you love cats?
+            <label>
+              <input type="radio" name="cat">
+                Yes
+            </label>
+            <label>
+              <input type="radio" name="cat">
+                No
+            </label>
+          </div>
+          <label class="form-field">
+            What is your favorite color?
+            <input type="color" name="color">
+          </label>
+          <label class="form-field">
+            What time do you go to bed?
+            <input type="time" name="time">
+          </label>
+          <label class="form-field">
+            What are your favorite brand of cars?
+            <select name="cars" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+          <label class="form-field">
+            How do you rate our work?
+            <input type="range" name="rate">
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>Additional info:</legend>
+          <label class="form-field">
+            Comments:
+            <textarea name="textarea"></textarea>
+          </label>
+          <label class="form-field">
+            Would you recommend us?
+            <select name="recommend">
+              <option value="Yes">yes</option>
+              <option value="No">no</option>
+            </select>
+          </label>
+        </fieldset>
+      </form>
+      <button type="submit" class="form">Submit</button>
```

Comment:
```suggestion
      <button type="submit" class="form">Submit</button>
      </form>
```

You made just one form for all field sets, but your submit button is out of this form, so it doesn't work, check it please